manifests/service.pp

```diff
@@ -1,4 +1,8 @@
 class mcollective::service {
+  if "aio_agent_version" in $facts and $facts["aio_agent_version"] =~ /^6/ {
```

Comment:
Probably [versioncmp](https://puppet.com/docs/puppet/5.5/function.html#versioncmp) should be used instead of `=~`.

Reply:
yeah good call, I'll move to that as i work ont his stuff